src/Layer/OGC3DTilesLayer.js

```diff
@@ -167,7 +169,58 @@ class OGC3DTilesLayer extends GeometryLayer {
         this.tilesRenderer.dispose();
     }
 
-    // TODO Methods: attach; detach; getObjectToUpdateForAttachedLayers; getC3DTileFeatureFromIntersectsArray?
+    // TODO: seriously rename that method
+    /**
+     * Get the attributes for the closest intersection from a list of
+     * intersects.
+     * @param {THREE.Intersection<THREE.Mesh | THREE.Points>[]} intersects
+     * @returns {Record<string, any> | null}
+     */
+    getC3DTileFeatureFromIntersectsArray(intersects) {
+        if (!intersects.length) { return null; }
+
+        const { face, index, object } = intersects[0];
+
+        /** @type{number|null} */
+        let batchId;
+        if (object.isPoints && index) {
+            batchId = index;
+        } else if (object.isMesh && face) {
+            batchId = object.geometry.getAttribute('_BATCHID').getX(face.a);
+        } else {
+            return null;
+        }
+
+        let tileObject = object;
+        while (!tileObject.batchTable) {
+            tileObject = tileObject.parent;
+        }
+
+        return tileObject.batchTable.getDataForId(batchId);
+    }
+
+    /**
+     * Get all 3D objects (mesh and points primitives) as intersects at the
+     * given non-normalized screen coordinates.
+     * @param {View} view
+     * @param {THREE.Vector2} coords
+     * @param {number} radius
+     * @param {THREE.Intersection<THREE.Mesh | THREE.Points>[]} [target=[]]
+     * @returns {THREE.Intersection<THREE.Mesh | THREE.Points>[]}
+     */
+    pickObjectsAt(view, coords, radius, target = []) {
+        const camera = view.camera.camera3D;
+        _raycaster.setFromCamera(view.viewToNormalizedCoords(coords), camera);
+        _raycaster.near = camera.near;
+        _raycaster.far = camera.far;
+
+        _raycaster.firstHitOnly = true;
```

Comment:
Are we sure we only want the first object only? Can you make it configurable instead?

Reply:
This was for efficiency purpose. Since the intersects are already sorted by distance (since this is a ray) and that we only use the closest intersects in `getC3DTileFeatureFromIntersectsArrays` (old and new implementation), I thought that we could stop at the first intersection.

Reply:
I think there is use cases where `pickObjectsAt` is used without `getC3DTileFeatureFromIntersectsArray` (either directly on the layer or on the `View` with `View.pickObjectsAt`) and where one would like to have all the intersections returned. We can make it configurable and default `firstHitOnly` to `true`, WDYT?

Reply:
Agree, but we'll have to modify the signature of `picksObjectAt`.

Reply:
I propose to remove it for now and test on larger datasets if there is a huge overhead.

Reply:
> Agree, but we'll have to modify the signature of `picksObjectAt`.

Yes you're right, that's another subject. Let's keep the optim for now and see if we have a usecase for making it configurable later.